test/requirements.in

```diff
@@ -18,3 +18,5 @@ black
 isort
 flake8
 torch
+httpx
```

Comment:
This is just test dependencies, if a lib is used within the Gradio library, you also need to add them to the setup.py and requires.txt

Reply:
Oh, I forgot that. Now, it's fixed 👍